pkg/runctl/log/custom/pipelinerun_accessor.go

```diff
@@ -0,0 +1,48 @@
+package custom
+
+import (
+	"github.com/SAP/stewardci-core/pkg/apis/steward/v1alpha1"
+	yaml "gopkg.in/yaml.v3"
+)
+
+// LoggingDetailsProvider extracts details from a pipeline run to be added to the log
+type LoggingDetailsProvider func(run *v1alpha1.PipelineRun) []any
+
+// ParseLoggingDetailsProvider returns a list of LoggingDetailsProviders created from a yaml string.
+func ParseLoggingDetailsProvider(strVal string) ([]LoggingDetailsProvider, error) {
```

Comment:
As said above, the return value should be a _single_ function that provides the logging attributes from _all_ accessors.

```suggestion
func ParseLoggingDetailsConfig(strVal string) (LoggingDetailsProvider, error) {
```